frontend/src/HomePage.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import { useRef } from "react";
```

Comment:
You're not using this, keep your code clean removing unused variables, libraries, etc.